src/gui/advancedsettings.cpp

```diff
@@ -537,6 +540,9 @@ void AdvancedSettings::loadAdvancedSettings()
             + ' ' + makeLink("https://www.libtorrent.org/reference-Settings.html#validate_https_trackers", "(?)"))
             , &m_checkBoxValidateHTTPSTrackerCertificate);
 #endif
+    // Disallow connection to peers on privileged port
+    m_checkBoxBlockPeersOnPrivilegedPorts.setChecked(session->blockPeersOnPrivilegedPorts());
+    addRow(BLOCK_PEERS_ON_PRIVILEGED_PORTS, (tr("Disallow connection to peers on privileged port") + ' ' + makeLink("https://libtorrent.org/single-page-ref.html#no_connect_privileged_ports", "(?)")), &m_checkBoxBlockPeersOnPrivilegedPorts);
```

Comment:
@NotTsunami:  English grammar correction: 

> Disallow connection to peers on privileged port 

…should be… 

> Disallow connection to peers on privileged **ports** 

The double plural is necessary because the two nouns are on different levels of the statement.